pixi/src/ui/report/CoreWebVitalsReportView.js

```diff
@@ -28,9 +28,10 @@ class WeightedScale {
   }
 
   render(data, unit) {
-    this.pitch.style.left = `${
+    this.pitch.style.left = `${Math.min(
```

Comment:
Ok for now, but I discussed with Ferdi how we should display the limit in the frontend and we think we should use the arrow here. But that is something we would introduce in an other PR.

Reply:
Sounds good, yes that will be a good change to make. Thanks.